fixtures/consumer/build.test.ts

```diff
@@ -34,7 +34,7 @@ describe('build', () => {
     // eslint-disable-next-line no-console
     console.log('Running `pnpm build`...');
     await promisify(exec)('pnpm build');
-  });
+  }, 60_000);
```

Comment:
Was this for debugging, or is it needed for something im missing?

Reply:
I noticed it was sometimes timing out. Maybe it was just my machine, but the default 30 seconds. I can't seem to find an instance of it failing in CI...